requirements.txt

```
@@ -6,8 +6,9 @@ cachetools==2.0.1
 configargparse==0.13.0
 humanfriendly==4.16.1
 psycopg2-binary==2.7.5
-modelforge==0.12.0
+modelforge>=0.13.1,<0.14.0
```

Sorry, I missed this. We always pin versions in requirements.txt

```suggestion
modelforge==0.13.1
```

---

Good to know